gcs/doc.md

```diff
@@ -0,0 +1,147 @@
+# How to Use Bifrost with Google Cloud Storage (GCS)
+Welcome to the Bifrost documentation for Google Cloud Storage (GCS)! This guide will walk you through the steps of using Bifrost to upload files to GCS.
+
+## Overview
+Google Cloud Storage is a popular cloud storage service that allows you to store and access your data on Google's infrastructure. Bifrost provides a simple and intuitive way to upload files to GCS without having to write complex code.
+
+## Prerequisites
+Before you can start using Bifrost to upload files to Google Cloud Storage, you'll need to make sure you have the following:
+- A Google Cloud account with GCS access
+- A GCS bucket to upload files to
+- Google Cloud SDK installed on your local machine
+- Bifrost installed on your local machine
+
+### Steps:
+#### Create a GCS bucket
+1. Login to your Google Cloud account and navigate to the GCS console
+2. Click on the "Create bucket" button and follow the prompts to create a new bucket
+_Note the name of your bucket as you will need it later_
+
+#### Set up Google Cloud credentials
+1. Create a new service account by navigating to the IAM & Admin console and selecting "Service accounts" from the left-hand menu
+2. Click on the "Create Service Account" button and follow the prompts to create a new service account
+3. Once you have created the service account, navigate to the "Keys" tab and click on the "Create Key" button
+4. Select "JSON" as the key type and download the JSON key file
+_Note the path to your JSON key file as you will need it later_
+
+#### Install Google Cloud SDK
+1. Download and install the Google Cloud SDK from the official website
+2. Follow the prompts to set up the SDK on your local machine
+3. Authenticate the SDK using the command `gcloud auth login`

```

Comment:
Hi...I do understand that this covers gcloud setup but we don't want to be covering all that setup in here. 
We really just want to focus on selling the solution of Bifrost and that means we have to be as concise as possible. 

So, please, we'll have to take away lines 14 to 30.